### PR TITLE
Zqs 1255/refactor cirq tests

### PR DESCRIPTION
**Pull request by Boniface316**
## Description

Include description of feature this PR introduces or a bug that it fixes. Include the following information:

Single file is used to perform test on both qsim and cirq.

## Please verify that you have completed the following steps

- [X] I have self-reviewed my code.
- [X] I have included test cases validating introduced feature/fix.

### CODE REVIEW

File: src/orquestra/integrations/cirq/conversions/_cirq_pauli_conversions.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 def pauliop_to_cirq_paulisum(
     pauli_operator: PauliRepresentation,
-    qubits: Union[List[cirq.GridQubit], List[cirq.LineQubit]] = None,
+    qubits: Union[List[cirq.GridQubit], List[cirq.LineQubit], None] = None,
```
[the remark this answers is not in the document]
Comment (Boniface316) [reply src/orquestra/integrations/cirq/conversions/_cirq_pauli_conversions.py:12]: Done

File: src/orquestra/integrations/cirq/simulator/simulator.py [hunk 2]
```diff
@@ -33,7 +34,7 @@ class CirqSimulator(CirqBasedSimulator):
     def __init__(
         self,
         noise_model: cirq.NOISE_MODEL_LIKE = None,
-        seed: int = None,
+        seed: Union[int, None] = None,
```
[the remark this answers is not in the document]
Comment (Boniface316) [reply src/orquestra/integrations/cirq/simulator/simulator.py]: Done

File: tests/orquestra/integrations/cirq/simulator/simulator_test.py [hunk 3]
```diff
@@ -4,60 +4,74 @@
...
+    ],
+)
+def simulator(request):
+    return request.param
 
-@pytest.fixture()
-def wf_simulator():
-    return CirqSimulator()
 
+class TestCirqSimulator:
```
[the remark this answers is not in the document]
Comment (Boniface316) [reply tests/orquestra/integrations/cirq/simulator/simulator_test.py]: Done

**Comment by mstechly** [comment]
Closing due to GitHub confusion, will reopen as a new one.

**Pull request closed without merging**